.dockerignore

```diff
@@ -0,0 +1,5 @@
+.git
+**/*.swp
+**/*.pyc
+.vagrant
+Vagrantfile
```

Comment:
Note that this file lists all the things that don't get copied over with a `COPY`. This doesn't affect mounting volumes at all.